src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -295,15 +295,28 @@ public FeedbackSessionQuestionsBundle getFeedbackSessionQuestionsForInstructor(
                         courseId, userEmail);
 
         InstructorAttributes instructorGiver = instructor;
+        Map<String, List<FeedbackResponseCommentAttributes>> commentsForResponses =
+                new HashMap<>();
+        CourseRoster roster = new CourseRoster(studentsLogic.getStudentsForCourse(courseId),
+                instructorsLogic.getInstructorsForCourse(courseId));
+
+        Map<String, String> emailNameTable = new HashMap<>();
+        Map<String, String> emailLastNameTable = new HashMap<>();
+        Map<String, String> emailTeamNameTable = new HashMap<>();
```

Comment:
As in combine them into one?

Reply:
Try to think if is necessary for `FeedbackSessionQuestionsBundle` to have these tables?

Reply:
I have removed emailLastNameTable, but I think other two tables are necessary because we use `FeedbackSessionQuestionsBundle` in `FeedbackSubmissionEditPageData`. These tables help in getting name/team from email (where email could be name of team) required to form comment. I checked  if roster could help in getting these information. Also this logic is taken `FeedbackSessionResultsBundle` which also have these tables and method `public String getNameForEmail(String email)`.